heron/schedulers/src/java/com/twitter/heron/scheduler/ecs/EcsContext.java

```diff
@@ -0,0 +1,60 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.twitter.heron.scheduler.ecs;
+
+import com.twitter.heron.spi.common.Context;
+
+
+/**
+ * Created by ananth on 4/29/17.
+ */
+public class EcsContext extends Context {
+  public static final String PART1 = "version: '2'\n"
+      + "services:\n"
+      + "  container_number:\n"
+      + "    image: ananthgs/onlyheronandubuntu\n";
+  public static final String CMD = "    command: [\"sh\", \"-c\", \"mkdir /s3; cd /s3 ;"
```

Comment:
This seems to be similar to what we had to do with the Marathon Docker environment. You can basically run the executor on any image built with the Dockerfile in the `docker/Dockerfile.dist.ubuntu<version>` image. The issue here is that we don't currently have a common Docker repo for this to live in yet. I'm thinking we need to have a `heron/executor` image with tagged versions that is created on new version releases. Otherwise, we're going to keep running into cases like this where people are building their own images, which is an extra step to deployment and a bit of a pain. Could also bundle in the core binaries on the release as well to eliminate the need to pull them when the Docker image is deployed.